File: src/main/java/com/rs/java/game/area/multi/CorporealBeastCaveMulti.java
package com.rs.java.game.area.multi;

import com.rs.java.game.area.Area;

public final class CorporealBeastCaveMulti extends Area {

	@Override
	public Area update() {
		return this;
	}

	@Override
	public String name() {
		return "Corporeal Beast Cave Multi";
	}

	public CorporealBeastCaveMulti() {
		super(11844, 11588);
	}

	@Override
	public boolean member() {
		return false;
	}

	@Override
	public Environment environment() {
		return Environment.MULTI;
	}

}

File: src/main/java/com/rs/kotlin/game/world/area/areas/multi/CorporealBeastCaveMulti.kt
package com.rs.kotlin.game.world.area.areas.multi

import com.rs.kotlin.game.world.area.Area

class CorporealBeastCaveMulti : Area(11844, 11588) {
    override fun update(): Area {
        return this
    }

    override fun name(): String {
        return "Corporeal Beast Cave Multi"
    }

    override fun member(): Boolean {
        return false
    }

    override fun environment(): Environment {
        return Environment.MULTI
    }
}
